### DISCUSSION TITLE
MD: term/definition recognizes date as initiation of numbered lists

### DISCUSSION THREAD

**Discussion by r-cheologist** [quarto-dev/quarto-cli#6541] [Q&A] [answered]
### Description

Following the [documentation](https://quarto.org/docs/authoring/markdown-basics.html#lists), I am defining term/definition lists like so:

```markdown
Date Format 1:
: 16^th^ August 2023

Date Format 2:
: 16.08.2023
```
When rendering this to `PDF`, the second version introduces unwanted horizontal and vertical whitespace, as the leading `16.` appears to be interpreted as initializing a list.

How can I avoid that?

**Comment by r-cheologist** [comment, marked as answer]
```markdown
Date Format 2:
: &ZeroWidthSpace;16.08.2023
```
works, but is an eye sore ...

**Comment by mcanouil** [comment]
What's wrong exactly with the PDF output?
<img width="1460" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/475f0676-7a8b-4199-817f-46b5737846b1">
